pyfiglet/__init__.py

```diff
@@ -143,7 +155,8 @@ def preloadFont(cls, font):
                 font_path = path
                 break
             else:
-                for location in ("./", SHARED_DIRECTORY):
+                shared_dir = find_shared_dir()
```

Comment:
You don't really need this variable.  It could have been put directly in the next line.